src/components/lib/blubberFormGenerator/FieldBase.js

```diff
@@ -44,9 +45,66 @@ export default class FieldBase {
 		this.__ModelKey = '';
 		this.__ModelPointer = null;
 		this.__HasDefaultValue = false;
+		if ( Field === null ) {
+			this._Field = { name: '"not set"' };
+		}
 	}
 
-	__lookForPropertyAtVueObject( IsTypeOrFunction ) {
+	static _getName( Field ) {
+		let Name;
+		if ( Field.hasOwnProperty( 'bind' ) === true ) {
+			Name = Field.bind;
+		} else {
+			Name = Field.name;
+		}
+
+		if ( typeof Name === 'undefined' ) {
+			return '"not set"';
+		} else {
+			return Name;
+		}
+	}
+
+	_validateIdentifier( Identifier ) {
+		let CurrentCharacter, Index;
+		if (
+			typeof Identifier !== 'string'
+		||
+			Utils.isEmpty( Identifier ) === true
+		) {
+			return false;
+		}
+
+		CurrentCharacter = Identifier.charCodeAt( 0 );
+		// First char should be a character
+		if (
+			(
+				CurrentCharacter < 65
+			||
+				CurrentCharacter > 90
+			)
+		&&
+			(
+				CurrentCharacter < 97
+			||
+				CurrentCharacter > 122
+			)
```

Comment:
maybe worth putting this condition to a private method with a meaningful name?
This way, you wouldn't get your eyes distracted by single "logical" line spanning over 10 rows, and also wouldn't need a comment

Reply:
It's now encapsulated in a new private method